### DISCUSSION TITLE
Welcome to angor Discussions!

### DISCUSSION THREAD

**Discussion by miladsoft** [block-core/angor#142] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oathar** [comment]
👋 Hi, I'm Athar, a Computer Science student. I have experience in HTML, CSS, JavaScript, Java, Python, Git/GitHub, React.js, Vue.js,  Node.js, SQL and Docker.  Looking forward to contributing to Enhance UI/UX of the Angor decentralized P2P Funding Protocol. 

@miladsoft I am not able to join your Discord. Kindly share a valid link.

Thank you
